deploy/packages/deb/debian/postinst

```diff
@@ -38,6 +35,12 @@ chmod -R 0755 /usr/lib/emqx/bin
 ln -s /usr/lib/emqx/bin/emqx /usr/bin/emqx
 ln -s /usr/lib/emqx/bin/emqx_ctl /usr/bin/emqx_ctl
 
+if pidof systemd; then
+    systemctl enable emqx;
```

Comment:
I am not sure if the name `emqx` is also correct for emqx-ee.

Reply:
It is not necessary since `sysmtectl enable emqx` will reload the systemd daemon.

reload systemd daemon in `postrm` looks dummy, I should remove it.

Reply:
> I  am not sure if the name emqx is also correct for emqx-ee.

checked the service file in  https://www.emqx.com/en/downloads/enterprise/4.3.5/emqx-ee-centos7-4.3.5-amd64.rpm, service name is `emqx` not `emqx-ee`